server/src/main/java/org/elasticsearch/rest/RestController.java

```diff
@@ -157,6 +189,10 @@ public void registerHandler(RestRequest.Method method, String path, RestHandler
 
     @Override
     public void dispatchRequest(RestRequest request, RestChannel channel, ThreadContext threadContext) {
+        if (tracerLog.isTraceEnabled() && shouldTraceRequest(request.uri())) {
+            tracerLog.trace(new ParameterizedMessage(
+                "Incoming request [{}][{}] on [{}]", request.method(), request.uri(), request.getHttpChannel()));
```

Comment:
can you show how this will look like? In particular, what does `request.getHttpChannel()` print?

Reply:
Sure:

```
[2020-01-31T15:22:15,288][TRACE][o.e.r.R.tracer           ] [runTask-0] Incoming request [GET][/_nodes/stats?pretty] on [Netty4HttpChannel{localAddress=/127.0.0.1:9200, remoteAddress=/127.0.0.1:37162}]
```